x-pack/plugins/apm/public/components/shared/kuery_bar/get_bool_filter.ts

```diff
@@ -35,6 +37,19 @@ export function getBoolFilter({
     });
   }
 
+  const environmentFilter = getEnvironmentEsField(urlParams.environment);
+  if (!isEmpty(environmentFilter) && environmentFilter['service.environment']) {
```

Comment:
Don't we have a separate function for this?

Reply:
I don't know, do we?

Reply:
yeah, it looks like the same, plus the must_not case, I'm going to move the `environmentQuery` to the common folder and use it here.

Reply:
I see that the `getEnvironmentEsField` is only used in the alert functions